va_explorer/va_logs/admin.py

```diff
@@ -1,3 +1,7 @@
 from django.contrib import admin
+from .models import VerbalAutopsy, Location, CauseOfDeath
 
 # Register your models here.
+admin.site.register(VerbalAutopsy)
```

Comment:
Are these changes needed? If so, could you expand on what they do for this feature?

Reply:
no, removed